windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -19,6 +19,20 @@ You must have a subscription to receive support for the Red Hat WMCO. Deploying
 
 include::modules/wmco-prerequisites.adoc[leveloffset=+1]
 
+[id="wmco-2-0-4"]
+== Release notes for Red Hat Windows Machine Config Operator 2.0.4
+
+Issued: 2022-02-14
+
+The WMCO 2.0.4 is now available with bug fixes. The components of the WMCO were released in link:https://access.redhat.com/errata/RHSA-2022:0241[RHSA-2022:0241].
+
+[id="wmco-2-0-4-bug-fixes"]
+=== Bug fixes
+
+* A flaw was found in the way Unicode standards are implemented in the context of development environments, which have specialized requirements for rendering text. An attacker could exploit this to deceive a human reviewer by creating a malicious patch containing well placed BiDi characters. The special handling and rendering of those characters could have been used to hide unexpected and potentially dangerous behavior from the reviewer. This security issue has been fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2005819[**BZ#2005819**])
```

Comment:
A flaw was found in the way Unicode standards are implemented in the context of development environments, which have specialized requirements for rendering text.

"that have specialized"?? Unless it's all development environments.

Maybe: s/BiDi/bidirectional

s/well placed/well-placed per https://www.merriam-webster.com/dictionary/well-placed

Reply:
Should this have the same BZ number as the WMCO 1.0.6 release? Is one of these a backport with a different number?